styles/styles.css

```diff
@@ -56,6 +56,13 @@
     @apply p-2 font-semibold;
   }
 
+  img {
+    background-color: black;
+    border: 1px solid #ccc;
```

Comment:
Make sure to test this a lot, adding a border can affect a lot of images that you may not intend for it to affect...